dlio_benchmark/utils/utility.py

```diff
@@ -22,58 +22,66 @@
 from functools import wraps
 import threading
 import json
-from typing import Dict
-import pathlib
-from enum import Enum
+import tracemalloc
```

Comment:
why trace malloc?

Reply:
we import `tracemalloc` at the line 246. I just put this up since it is more readable.

apparently this is not needed and may be the remnant of previous experimentation.

maybe we should remove it? what do you think? cc @zhenghh04 as well

Reply:
Lets remove if not used.

Reply:
this is done